packages/flutter/lib/src/cupertino/thumb_painter.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/painting.dart';
+
+final MaskFilter _kShadowMaskFilter = new MaskFilter.blur(BlurStyle.normal, BoxShadow.convertRadiusToSigma(1.0));
+
+class CupertinoThumbPainter {
```

Comment:
Having this as a class like this is fine. I wouldn't bother making it a decoration for now.